python_examples/covidsim_comparison/gibraltar_inputs/microcell_conversion.py

```diff
@@ -0,0 +1,70 @@
+#
+# Converts gibraltar population data from covidsim into compatible input file
+#
+
+import math
+import numpy as np
+import pandas as pd
+
+import pyEpiabm as pe
+
+
+def min_separation(x_loc, y_loc):
+    """Returns minimum separation between any pair of locations"""
+    assert len(x_loc) == len(y_loc), "Mismatched coordinates"
+    sep = np.inf
+
+    for x1, y1 in zip(x_loc, y_loc):
+        for x2, y2 in zip(x_loc, y_loc):
+            if (x1 == x2) and (y1 == y2):
+                continue
+            if np.linalg.norm([x2-x1, y2-y1]) < sep:
+                sep = np.linalg.norm([x2-x1, y2-y1])
+    return sep
+
+
+pe.Parameters.set_file("python_examples/covidsim_comparison/"
+                       + "gibraltar_parameters.json")
+
+mcell_num = 81
+file_path = "python_examples/covidsim_comparison/gibraltar_inputs/"
```

Comment:
Maybe Lizzi means we should consider using the `os.path.join(os.path.dirname(__file__), ...` to make it less sensitive to where the file is run from

Reply:
Makes sense, happy to make that change in the workflows (although I can't see why you wouldn't be running them from the top level), and will also edit the readme!